nbclient/tests/test_client.py

```diff
@@ -201,10 +204,10 @@ def normalize_output(output):
         if isinstance(value, string_types):
             output['data'][key] = normalize_base64(value)
     if 'traceback' in output:
-        tb = [
-            re.sub(ipython_input_pat, '<IPY-INPUT>', strip_ansi(line))
-            for line in output['traceback']
-        ]
+        tb = []
+        for line in output["traceback"]:
+            line = re.sub(ipython_input_pat, '<IPY-INPUT>', strip_ansi(line))
+            line = re.sub(ipython8_input_pat, '<IPY-INPUT>', strip_ansi(line))
         output['traceback'] = tb
```

Comment:
`tb` is always empty here?

Reply:
I don't understand the question. The previous version created the content of `tb` as a list comprehension so the variable was and still is initiated here as a list of lines.

Reply:
But I only see `tb = []`, or am I missing something?

Reply:
Ah, I'm sorry, copy-paste error. Fixed. Thanks!